src/platform/site-wide/sass/modules/_m-vet-nav.scss

```diff
@@ -248,7 +248,7 @@ body.va-pos-fixed {
 }
 
 .vetnav-level2 {
-  background: $color-primary-darkest url(/img/arrow-right-white.svg) right 3rem
+  background: $color-primary-darkest url(/img/arrow-right-white.svg) right 2rem
```

Comment:
What does this change? Does it affect anything in desktop view?

Reply:
After adding the `padding-right` attribute to the `.mm-link-container` class, the `arrow-right-white.svg` shifted to the left. This keeps `arrow-right-white.svg` in place so it's position remains the same.

 <details>
  <summary>Screenshot</summary>
<img width="360" alt="Screen Shot 2020-07-14 at 9 02 03 AM" src="https://user-images.githubusercontent.com/9042882/87449161-abc63b80-c5b1-11ea-953d-6b96b6a15bc5.png">
</details>

Reply:
That makes sense.

Still, I'm a bit uncomfortable that we're mixing `rem` and `px` units here. I'd rather see `rem` in more places, but I don't know the full impact of changing both of the paddings to use that for `.mm-link-container`.

Reply:
Maybe this calls for a conversation with the team on updating `vet-nav` to use primarily `rem`.